support older ssp (gcc-3.4.x).  this ssp was never in mainline gcc, thus
upstream wine wont accept it.  can drop once hardened gets a stable gcc-4.x.

http://bugs.gentoo.org/66002

--- loader/preloader.c
+++ loader/preloader.c
@@ -155,6 +155,8 @@ struct wld_link_map {
 
 /* similar to the above but for -fstack-protector */
 void *__stack_chk_guard = 0;
+void *__guard = 0;
+void __stack_smash_handler(void) { return; }
 void __stack_chk_fail(void) { return; }
 
  * The _start function is the entry and exit point of this program
